rules/windows/discovery_privileged_localgroup_membership.toml

```
@@ -14,13 +14,67 @@ index = ["winlogbeat-*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Enumeration of Privileged Local Groups Membership"
-note = """## Config
+note = """## Triage and analysis
 
-This will require Windows security event 4799 by enabling audit success for the Windows Account Management category and
-the Security Group Management subcategory.
+### Investigating Enumeration of Privileged Local Groups Membership
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+After a successful initial compromise, attackers may try to gain situational awareness of the environment to plan their
+next steps. This can happen by running commands to enumerate network resources, users, connections, files, and security
+software installed.
+
+This rule looks for the enumeration of privileged local groups membership by suspicious processes, excluding known legit
```

Combines @Mikaayenson and @nastasha-solomon's suggestions, plus my own "legit" revision. I'm not sure if there's just one or multiple "groups" but this is assuming it's plural:
```suggestion
This rule looks for the enumeration of privileged local groups' membership by suspicious processes, and excludes known legitimate
```